packages/component/src/SendBox/SuggestedAction.js

```diff
@@ -81,7 +78,13 @@ const SuggestedAction = ({
   return (
     <div
       aria-hidden={ariaHidden}
-      className={classNames('webchat__suggested-action', rootClassName, suggestedActionStyleSet + '')}
+      className={classNames(
+        'webchat__suggested-action',
+        { 'webchat__suggested-action--rtl': direction === 'rtl' },
```

Comment:
Will this cause behavior change? I seem to recall that we didn't need to flip for RTL but I could be wrong.

Reply:
The current layout in RTL was not exactly right, the padding was too thin because we put the padding on the wrong side under RTL.

In the screenshot below, the green indicates the edge of the UI. And the padding on the right hand side was too narrow because we accidentally put it on the left (ignoring RTL).

So the change should be good.

![image](https://user-images.githubusercontent.com/1622400/103040864-94f51f00-4529-11eb-92ed-b576861d0cd1.png)

(From https://github.com/microsoft/BotFramework-WebChat/blob/master/__tests__/__image_snapshots__/chrome-docker/rtl-js-rtl-ui-should-show-suggested-actions-with-images-1-snap.png)